server/athenian/api/internal/miners/jira/issue.py

```diff
@@ -814,27 +817,12 @@ def _validate_and_clean_issues(df: pd.DataFrame, acc_id: int) -> pd.DataFrame:
 class PullRequestJiraMapper:
     """Mapper of pull requests to JIRA tickets."""
 
-    @classmethod
-    @sentry_span
-    async def append(
-        cls,
-        prs: PullRequestFactsMap,
-        entities: JIRAEntityToFetch | int,
-        meta_ids: tuple[int, ...],
-        mdb: DatabaseLike,
-    ) -> None:
-        """Load and insert "jira_id" to the PR facts."""
-        pr_node_ids = defaultdict(list)
-        for node_id, repo in prs:
-            pr_node_ids[node_id].append(repo)
-        jira_map = await cls.load(pr_node_ids, entities, meta_ids, mdb)
-        for pr_node_id, jira in jira_map.items():
-            for repo in pr_node_ids[pr_node_id]:
-                try:
-                    prs[(pr_node_id, repo)].jira_ids = jira
-                except KeyError:
-                    # we removed this PR in JIRA filter
-                    continue
+    _column_map = {
+        Issue.key.name: (PullRequestJIRADetails.ids.__name__, object),
+        Issue.project_id.name: (PullRequestJIRADetails.projects.__name__, "S"),
+        Issue.priority_id.name: (PullRequestJIRADetails.priorities.__name__, "S"),
+        Issue.type_id.name: (PullRequestJIRADetails.types.__name__, "S"),
```

Comment:
We must set the exact sizes after `S`, they are critical for performance (to avoid repacking). Can load them from `Issue.column.info["dtype"]`